spec/features/cart/show_spec.rb

```diff
@@ -75,5 +75,36 @@
 
       expect(page).to_not have_content("#{@itemB.name}: 5")
     end
+
+    it "I see a login prompt if I'm not logged in" do
+      expect(page).to have_content("You must login or register to check out.")
+    end
+
+    it "gives me a check out option if I'm logged in" do
+      click_on "LogIn"
+      fill_in "email", with: @user.email
+      fill_in "password", with: @user.password
+      click_on "Log In"
+
+      visit '/cart' #TECHNICAL DEBT LOL
```

Comment:
I replaced the session new with a redirect_back, and it didn't work, technically (the code did not to what it was supposed to but it did do what the spec says it's supposed to) so I just threw this in there. What that means for the user is that if they login from their cart they get redirected to their profile and have to click their cart again, which is obnoxious.